zimui/src/components/playlist/panel/PlaylistPanel.vue

```diff
@@ -51,10 +54,35 @@ const scrollToCurrentVideo = () => {
 }
 
 const emit = defineEmits(['shuffle', 'loop', 'hide-panel'])
+
+const items = computed(() => props.playlist.videos.slice(0, 48))
+
+const loadMoreItems = async () => {
+  return new Promise<VideoPreview[]>((resolve) => {
+    setTimeout(() => {
+      resolve(props.playlist.videos.slice(items.value.length, items.value.length + 12))
+    }, 100)
+  })
+}
+
+const load = async ({ done }: { done: (status: 'ok' | 'empty') => void }) => {
+  const res = await loadMoreItems()
```

Comment:
Same remark about `res`

Reply:
Changed it in [d1930a0](https://github.com/openzim/youtube/pull/289/commits/d1930a05040e8a51b581c2be289d366f87a93806)